test/imputors/substitute.jl

```diff
@@ -102,6 +102,12 @@
         )
         @test result == expected
     end
+
+    @testset "all missing" begin
+        a = [missing 2; missing 3]
+        # Test that substitution along columns with all missing data will just return that data.
+        Impute.substitute(a; dims=2) == a
```

Comment:
```suggestion
        @test Impute.substitute(a; dims=2) == a
```

Do you also want to test the error message?